src/Compilers/Core/Portable/Operations/CommonConversion.cs

```diff
@@ -16,11 +16,11 @@ public struct CommonConversion
         private enum ConversionKind
         {
             None = 0,
-            Exists =        1 << 0,
-            IsIdentity =    1 << 1,
-            IsNumeric =     1 << 2,
-            IsReference =   1 << 3,
-            IsImplicit =    1 << 4,
+            Exists = 1 << 0,
+            IsIdentity = 1 << 1,
+            IsNumeric = 1 << 2,
+            IsReference = 1 << 3,
+            IsImplicit = 1 << 4,
```

Comment:
Seems worse.

Reply:
➡️ By design. Our .editorconfig is not configured to ignore these spaces.